lib/actions/reassemble.js

```diff
@@ -18,12 +28,13 @@ async function timer(this_) {
 
     // eslint-disable-next-line no-await-in-loop
     await this_.emit('data', messages.newMessageWithBody({
-      groupSize: results.messages.length,
+      groupSize: Object.keys(results.messageIdsSeen).length,
```

Comment:
Why is this change needed?

Reply:
In the previous version, the messageIdSeen was empty, so we used the messages.length array to count the number of element present in the group. But now that the messageIdsSeen(randon gui) is always defined, we used it to know the exact number of elements present in the group.

Reply:
:+1: